src/tasks/tasks.py

```diff
@@ -792,11 +802,11 @@ def __init__(self, path, max_seq_len, name, probe_path="probe_dummy.tsv",
     def load_data(self, path, max_seq_len, probe_path):
         targ_map = {'neutral': 0, 'entailment': 1, 'contradiction': 2}
         tr_data = load_tsv(os.path.join(path, 'train_dummy.tsv'), max_seq_len,
-                           s1_idx=1, s2_idx=2, targ_idx=None, targ_map=targ_map, skip_rows=0)
+                           s1_idx=1, s2_idx=2, has_labels=False, targ_map=targ_map, skip_rows=0)
```

Comment:
why does the training data not have labels?

Reply:
The previous implementation for this task didn't return any labels as well- let me check for NPS why there are no labels for train though.

Reply:
Actually,I'm honestly not sure what NPS is - I can't find it in download_data_glue